spec/models/miq_schedule_spec.rb

```diff
@@ -27,29 +27,108 @@
       FactoryBot.create(:miq_schedule, :updated_at => 1.year.ago, :filter => miq_expression, :sched_action => sched_action, :userid => user.userid, :last_run_on => Time.zone.now)
     end
 
+    let(:miq_automate_schedule) do
```

Comment:
Does this belong under `context "AutomationTask"`?  That's the only consumer that I see